src/extra/holba_auxiliaryScript.sml

```diff
@@ -96,10 +96,9 @@ QED
 
 
 Theorem INDEX_FIND_EQ_NONE:
-  !l i P. (INDEX_FIND i P l = NONE) <=> (~(EXISTS P l))
+  !l i P. INDEX_FIND i P l = NONE <=> ~EXISTS P l
```

Comment:
```suggestion
  !l i P. (INDEX_FIND i P l = NONE) <=> ~EXISTS P l
```
The first parenthesis pair here are actually helpful for readability.

Reply:
Not sure I agree, `<=>` only operates on Booleans so there can be no ambiguity regarding the `NONE`.